api/src/main/java/io/strimzi/api/kafka/model/topic/KafkaTopicStatus.java

```diff
@@ -37,4 +38,13 @@ public String getTopicName() {
     public void setTopicName(String topicName) {
         this.topicName = topicName;
     }
+
+    @Description("Topic id")
```

Comment:
We're exposing this to client applications. We should try to define what it means so such applications can use it correctly. This is a bit tricky because we have the possibility of conflicting KafkaTopics. I think this is OK:

```suggestion
    @Description("The topic's id. For a KafkaTopic with the ready condition, this will change only if the topic gets deleted and recreated with the same name.")
```